src/libraries/System.IO.FileSystem/tests/FileStream/FileStreamConformanceTests.Windows.cs

```diff
@@ -176,4 +176,148 @@ public struct SHARE_INFO_502
         [DllImport(Interop.Libraries.Netapi32)]
         public static extern int NetShareDel([MarshalAs(UnmanagedType.LPWStr)] string servername, [MarshalAs(UnmanagedType.LPWStr)] string netname, int reserved);
     }
+
+    [PlatformSpecific(TestPlatforms.Windows)] // the test setup is Windows-specifc
+    [OuterLoop("Has a very complex setup logic that in theory might have some side-effects")]
+    [ConditionalClass(typeof(PlatformDetection), nameof(PlatformDetection.IsNotWindowsNanoServer))]
```

Comment:
Why did you decide to exclude Windows Nano?

Reply:
Nano is missing a lot of `.dll` and device interfaces don't work very well with it: https://social.technet.microsoft.com/Forums/ie/en-US/e982d40c-2f4d-4a41-a310-d47cd3430bcc/setupdienumdeviceinfo-not-working-in-nano-server?forum=NanoServer